cannawen/metric_units_reddit_bot
# Add lint-staged and husky

## PR Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/cannawen/metric_units_reddit_bot/blob/master/CONTRIBUTING.md)
- [x] New functionality (or bug fix) is appropriately tested


## Brief description of changes:

I've added [husky](https://github.com/typicode/husky) and [lint-staged](https://github.com/okonet/lint-staged) along with an additional rule that eliminates the need for the `pre-commit` file.

## Verification

1. Check out the PR.
2. Inside a test, make one `it.only` and add a `console.log`:
    ```js
   context('Good bot', () => {
      it.only('should reply', () => {
        console.log('Hello!');
        verify("good bot", "Good human");
        verify("You are such a good robot", "Good human");
      });
    });
    ```

3. Save it and try to commit it. You should be unable due to the failing precommit hook.
